src/Services/SocialHistoryService.php

```diff
@@ -25,7 +25,7 @@ class SocialHistoryService extends BaseService
     public function __construct()
     {
         parent::__construct(self::TABLE_NAME);
-        $this->getUuidRegistry()->createMissingUuids();
+        UuidRegistry::createMissingUuidsForTables([self::TABLE_NAME]);
```

Comment:
oops, will revert this change

Reply:
@adunsulag , or should I keep this change and drop the getUuidRegistry() function in this class?

Reply:
ended up keeping this change and dropping the getUuidRegistry() function :)

Reply:
@bradymiller I think that's fine. sorry for the late response.  Having 'fun' with vitals.

Reply:
Need these? Also, i'm thinking we could automate interpretations from vital item ranges set in lists eg. heart rate 60-80 Normal etc.

    "2.16.840.1.113883.5.83": {
        name: "HL7 Result Interpretation",
        uri: "http://hl7.org/codes/ResultInterpretation#",
        table: {
            "B": "better",
            "D": "decreased",
            "U": "increased",
            "W": "worse",
            "<": "low off scale",
            ">": "high off scale",
            "A": "Abnormal",
            "AA": "abnormal alert",
            "H": "High",
            "HH": "high alert",
            "L": "Low",
            "LL": "low alert",
            "N": "Normal",
            "I": "intermediate",
            "MS": "moderately susceptible",
            "R": "resistant",
            "S": "susceptible",
            "VS": "very susceptible",
            "EX": "outside threshold",
            "HX": "above high threshold",
            "LX": "below low threshold",

        }
    }

Reply:
These are the codes that FHIR requires which are similar to what you have listed: http://hl7.org/fhir/R4/valueset-observation-interpretation.html